client/src/components/Tooltip/Tooltip.js

```diff
@@ -56,14 +51,14 @@ class Tooltip extends Component {
     document.removeEventListener('keydown', this.escFunction, false);
   }
 
```

Comment:
this is just vanilla javascript for adding/removing event listeners. 

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener

I mainly added it to detect if esc is pressed and it would close the tooltip because that is kind of what I would expect the behavior.